features/guidance/anticipate-lanes.feature

```diff
@@ -623,40 +623,7 @@ Feature: Turn Lane Guidance
             """
                  /a\
             x – b   d – y
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                |   |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                 | |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
-                  |
+                 \ /
```

Comment:
Well, here we still reduce from 640 meters to 200 meters, below from 200 to 150 meters